sdk/exportentities/template.go

```diff
@@ -37,3 +37,25 @@ func NewTemplate(wt sdk.WorkflowTemplate) (Template, error) {
 
 	return exportedTemplate, nil
 }
+
+// GetTemplate returns a sdk.WorkflowTemplate.
+func (w Template) GetTemplate() sdk.WorkflowTemplate {
+	wt := sdk.WorkflowTemplate{
+		Slug: w.Slug,
+		Name: w.Name,
+		Group: &sdk.Group{
+			Name: w.Group,
+		},
+		Description: w.Description,
+	}
+
+	for _, p := range w.Parameters {
```

Comment:
I think you can initialise your slice with right length/capacity

Reply:
done